predicates/content/content_length_between.go

```diff
@@ -0,0 +1,47 @@
+package content
+
+import (
+	"github.com/zalando/skipper/predicates"
+	"github.com/zalando/skipper/routing"
+	"net/http"
+)
+
+type contentLengthBetweenSpec struct{}
+
+type contentLengthBetweenPredicate struct {
+	min int64
+	max int64
+}
+
+// NewContentLengthBetween creates a predicate specification,
+// whose instances match content length header value in range from min to max inclusively.
+// example: ContentLengthBetween(0, 5000)
```

Comment:
We need to think about UX here.
Usually interval knobs use left-closed, right-open intervals (in math notation `[left, right)`) for convenient splitting like:
```
r1: ContentLengthBetween(0, 5000) -> ... ;
r2: ContentLengthBetween(5000, 10000) -> ... ;
r3: ContentLengthBetween(10000, 100000) -> ... ;
r0: * -> ... // matches Content-Length >= 100000 
```

I think this is how https://github.com/zalando/skipper/blob/master/docs/reference/predicates.md#between works (although undocumented)

Reply:
Another question if we should also add `ContentLengthMin` and `ContentLengthMax` (or `ContentLengthBetween` with a single argument) but I do not have convincing arguments for it at the moment - it looks like proposed predicate covers all the possible cases.